package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Any particular reason you chose to stick to `npm` instead of using `yarn` to install the dependencies (based on the lockfile above)?

Reply:
I was trying to rock the boat at least as possible 😅 I assumed (perhaps wrongly) that Yarn wouldn't be available in Travis and noticed that the Contributing guidelines made reference to `npm install` so stuck with NPM.